src/bp/lang-server/monitoring.ts

```diff
@@ -0,0 +1,63 @@
+import _ from 'lodash'
+import ms from 'ms'
+import onHeaders from 'on-headers'
+
+const debugMonitor = DEBUG('api:monitoring')
+
+let collectionEnabled = false
+let metrics = {}
+
+export const startMonitoring = () => {
+  console.log('Metrics collection enabled. Interval: ', process.env.MONITORING_INTERVAL)
+
+  setInterval(() => {
+    if (!metrics || !Object.keys(metrics).length) {
+      return
+    }
+
+    try {
+      debugMonitor(
```

Comment:
do we plan to send those metrics somewhere or we keep them in the std out ?

Reply:
No plan actually. Just wanted to have a quick view when it's first launched, but next step would be to make it use redis & store monitoring data the same was as BP server